src/main/java/com/github/kmu_shell_we/domain/mission/repository/SeasonMissionRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.github.kmu_shell_we.domain.mission.repository;
+
+import com.github.kmu_shell_we.domain.mission.entity.Mission;
+import com.github.kmu_shell_we.domain.mission.entity.SeasonMission;
+import com.github.kmu_shell_we.domain.season.entity.Season;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface SeasonMissionRepository extends JpaRepository<SeasonMission, UUID> {
+
+    SeasonMission findBySeasonAndMission(Season season, Mission mission);
+
+    List<SeasonMission> findAllSeasonMissionsBySeasonId(UUID seasonId);
```

Comment:
애초에 SeasonMission 레포지토리이기 때문에, findAll뒤에 대상체를 붙이지 않아도 됩니다.

또한, seasonId 대신 실질적인 season을 받아 처리해주세요.

```suggestion
    List<SeasonMission> findAllBySeason(Season season);
```